src/globals/test-extensions.php

```diff
@@ -33,14 +35,14 @@
 $no_strip = false;
 
 // compress with upx
-$upx = false;
+$upx = true;
```

Comment:
this should still default to false imo

Reply:
Just a test, no need to be sensitive to the file. We can go back in next PR.